appServer.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const PORT = 3000;
+
+const app = express();
+app.set("view engine", "hbs");
+app.set("views", "./views");
+app.use(express.json());
+
+app.use(express.static(__dirname + "/dist"));
```

Comment:
По заданию сервер должен сервить статику. Сервер-сайд рендеринг делать не нужно. То есть здесь должно быть только
```suggestion
app.use(express.static(__dirname + "/dist"));
```